devDocs/displayModel.md

```diff
@@ -0,0 +1,17 @@
+# Display model
+
+The display model (part of NVDA Helper), is a legacy mechanism for scraping
+information during the rendering of GUI applications.
+
+For information on working in this area see the `NVDAHelper/readme.md` file.
+
+## Testing GDI applications
+To test:
+- Use `NVDA+control+numPad 7` to enter screen review mode.
+- Use the number pad to read lines (7, 8, 9), read words, (4, 5, 6), or characters (1, 2, 3).
+
+Note:
+- For color reporting enable the option (via Document formatting panel) _"report color"_.

```

Comment:
This could be clearer, but note that for color reporting an option in "document formatting" needs to be enabled, for transparency reporting a second option in advanced settings needs to be enabled.

Reply:
Ah my mistake